ui/src/templates/containers/TemplatesIndex.tsx

```diff
@@ -26,7 +30,10 @@ type Props = WithRouterProps & StateProps
 @ErrorHandling
 class TemplatesIndex extends Component<Props> {
   public render() {
-    const {org, children} = this.props
+    const {org, children, flags} = this.props
+    if (flags.communityTemplates) {
```

Comment:
why pull all the flags in here and not just use the util function of `isFlagEnabled`?

Reply:
NOTE: to trigger re-render from the `connect` HOC

Reply:
let's think of a less ad-hoc way to do this so folks in the future don't have to rely on this, if possible